Tasks/PackerBuildV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -30,7 +30,7 @@
   "loc.input.help.customImageUrl": "Specify URL of base image. This will be used for installing pre-requisite(s) and application(s) before capturing machine image.",
   "loc.input.label.customImageOSType": "Base image OS",
   "loc.input.label.packagePath": "Deployment Package",
-  "loc.input.help.packagePath": "Specify the path for deployment package directory relative to $(System.DefaultWorkingDirectory). Supports minimatch pattern. Example path: FrontendWebApp/**/GalleryApp",
+  "loc.input.help.packagePath": "Specify the path for deployment package directory relative to $(System.DefaultWorkingDirectory). Supports minimatch pattern. Example path: FrontendWebApp/**/GalleryApp <br/> Please note that this package will be copied to temporary virtual machine which Packer creates. If the package contains large number of files and/or the files are very large in size, the upload can take quite long time(possibly running into few hours).",
```

Comment:
We should also suggest uploading to a temp azure storage and then downloading in packer script

Reply:
done